### PR TITLE
Update teams-powershell-move-from-sfbo.md

### PR DESCRIPTION
**Pull request by amareshbAtMicrosoft**
update the guidance on how to change from sfbo connector based scripts to microsoft teams module

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-move-from-sfbo.md [hunk 1]
```diff
@@ -16,17 +16,17 @@ appliesto:
 
 # Move from Skype for Business Online Connector to the Teams PowerShell module
 
-To move from using Skype for Business Online Connector to the Teams PowerShell module to manage Teams, you'll need to update your existing PowerShell scripts. This article explains how to do this.
+To move from using Skype for Business Online Connector to the Teams PowerShell module to manage Teams, you'll need to update existing PowerShell scripts. This article explains how to do this. Note that using Teams PowerShell module you can manage both Skype for Business Online and Teams users.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-move-from-sfbo.md:19]:
```suggestion
To move from using Skype for Business Online Connector to the Teams PowerShell module to manage Teams, you'll need to update existing PowerShell scripts. This article explains how to do this. Note that by using Teams PowerShell module you can manage both Skype for Business Online and Teams users.
```

File: Teams/teams-powershell-move-from-sfbo.md [hunk 2]
```diff
@@ -16,17 +16,17 @@ appliesto:
...
 1. Install the latest Teams PowerShell module. For steps, see [Install Microsoft Teams Powershell](teams-powershell-install.md).
-2. Uninstall Skype For Business Online Connector. To do this, in Control Panel, go **Programs and Features**, select **Skype for Business Online, Windows PowerShell Module**, and then select **Uninstall**. 
-3. In your PowerShell scripts, change the module name that's referenced in ```Import-Module``` from 
-```SkypeOnlineConnector``` or ```LyncOnlineConnector``` to ```MicrosoftTeams```.
-
-    For example, change ```Import-Module -Name SkypeOnlineConnector``` to ```Import-Module -Name MicrosoftTeams```.
-
-> [!NOTE]
-> Admins should continue to use [New-CsOnlineSession](https://docs.microsoft.com/powershell/module/skype/new-csonlinesession) and import the session before using Skype for Business Online cmdlets. 
+2. Uninstall Skype For Business Online Connector. To do this, in Control Panel, go **Programs and Features**, select **Skype for Business Online, Windows PowerShell Module**, and then select **Uninstall**.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-move-from-sfbo.md:22]:
```suggestion
2. Uninstall Skype For Business Online Connector. To do this, in Control Panel, go to **Programs and Features**, select **Skype for Business Online, Windows PowerShell Module**, and then select **Uninstall**.
```

File: Teams/teams-powershell-move-from-sfbo.md [hunk 3]
```diff
@@ -16,17 +16,17 @@ appliesto:
...
+3. 
+   | Original                              | Change To                     | Additional Comments                                             |
+   |-------------------------------------- |-------------------------------|-----------------------------------------------------------------|
+   | Import-Module SkypeOnlineConnector    | Import-Module MicrosoftTeams  |                                                                 |
+   | Import-Module LyncOnlineConnector     | Import-Module MicrosoftTeams  |                                                                 |
+   | $session = New-CsOnlineSession <br> Import-PsSession -Session $session    | Connect-MicrosoftTeams        | using New-CsOnlineSession and importing session is optional at this time and will be deprecated in the future                  |
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-move-from-sfbo.md:28]:
```suggestion
   | $session = New-CsOnlineSession <br> Import-PsSession -Session $session    | Connect-MicrosoftTeams        | Using New-CsOnlineSession and importing session is optional at this time and will be deprecated in the future.                  |
```

File: Teams/teams-powershell-move-from-sfbo.md [hunk 4]
```diff
@@ -16,17 +16,17 @@ appliesto:
...
+   | Enable-CsOnlinesessionForReconnection | -Remove it-                   | reconnections are now made automatic and this cmdlet is no longer needed and thus removed from module  |
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-move-from-sfbo.md:29]:
```suggestion
   | Enable-CsOnlinesessionForReconnection | -Remove it-                   | Reconnections are now made automatically and this cmdlet is no longer needed, thus it is removed from the module.  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@pupara PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Kateyanne** [comment]
Per Paneet Please disregard these changes as we made other changes and these are no longer required.

**Pull request closed without merging**